### PR TITLE
:bug: Exceptions Topic --- Add missing `"` :microscope: 

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Add a `"`

### Why

It was missing.

### CODE REVIEW

File: site/topics/sorting/sorting.rst [hunk 1]
```diff
@@ -70,7 +70,7 @@ Selection Sort
 .. admonition:: Activity
     :class: activity
 
-    Perform a insertion sort, with pencil and paper, on the list ``[3,7,4,1,5,2]``. Keep track of both the unsorted list
```
Comment (twentylemon) [inline site/topics/sorting/sorting.rst:73]: Forked from the wrong branch?

Comment (jameshughes89) [reply site/topics/sorting/sorting.rst]: Updating branch solved it.

**Review by twentylemon — APPROVED** [review]

**Comment by jameshughes89** [comment]
I tried it with quotes, like shown [here](https://github.com/actions/setup-python), but no luck.

**Review by twentylemon — APPROVED** [review]
:shipit: assuming #541 fixes the build here.

**Pull request merged**